deployment/spring-servlet-pa-tomcat/pom.xml

```diff
@@ -10,7 +10,13 @@
   <properties>
     <camunda.version>7.16.0</camunda.version>
     <spring.version>5.3.10</spring.version>
-    <tomcat.version>9.0.43</tomcat.version>
+    <tomcat.version>9.0.52</tomcat.version>
+    <servlet-api.version>3.0.1</servlet-api.version>
+    <h2.version>1.4.190</h2.version>
+    <slf4j.version>1.7.26</slf4j.version>
```

Comment:
See latest versions above for H2 and SLF4J

Reply:
👍